### PR TITLE
Garlic Lemon Pork Bites

### PR DESCRIPTION
**Pull request by Jokingg**
Made these for dinner tonight. Absolutely magical 'glow up' for pork. Cheesy, creamy, tangy deliciousness

### CODE REVIEW

File: source/meat/garlicPorkBites.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+* 2lbs boneless pork chop (cut into 3/4 inch cubes)
+* ~1 tsp Onion Powder
+* ~1 tsp Kosher Salt
+* 1 tablespoon Olive Oil
+
+Sauce:
+
+* 1/2 stick butter
+* 2 (or let's be honest, 4) cloves of garlic (minced)
```
Comment (jennank) [inline source/meat/garlicPorkBites.rst:23]: When should this be added?

File: source/meat/garlicPorkBites.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+* 1/2 cup parmesan cheese (fresh grated is always best, cause it's gonna melt)
+* 1/2 a Lemon juice
+
+Steps
+-----
+
+#. Cube Pork into 3/4 inch cubes (consistent size helps with cooking)
+#. Place pork in mixing bowl, season with onion powder, salt + pepper
+#. In a large skillet (I use 12+ inch) heat olive oil
+#. Sear pork for ~3-5 minutes per side, mixing so it cooks evenly. Brown = flavor, so don't be afraid to let it get a bit roasty
```
Comment (jennank) [inline source/meat/garlicPorkBites.rst]: Approx. burner temp?

**Review by jennank — CHANGES_REQUESTED** [review]
This was delicious! Made it for dinner last night and could not get enough of it. A few minor questions, but overall looks great!

**Comment by bernardopacini** [comment]
@jennank @Jokingg Where do we stand with this PR?

**Review by jennank — APPROVED** [review]

**Pull request merged**